### PR TITLE
Update Japanese translation (Line 694-710)

### PR DESCRIPTION
**Pull request by yosshy**
/hold for japanese developer's review
/assign @atoato88
/cc @YuikoTakada @t-inu @s-kawamura-w664

### CODE REVIEW

File: translations/ja.json [hunk 1]
```diff
@@ -691,23 +691,23 @@
...
+	"This addon does not have an endpoint defined for the 'addons open' command.\nYou can add one by annotating a service with the label {{.labelName}}:{{.addonName}}": "このアドオンは 'addons open' コマンド用に定義されたエンドポイントがありません。\nサービスに {{.labelName}}:{{.addonName}} ラベルを付与することでエンドポイントを追加できます",
 	"This can also be done automatically by setting the env var CHANGE_MINIKUBE_NONE_USER=true": "これは環境変数 CHANGE_MINIKUBE_NONE_USER=true を設定して自動的に行うこともできます",
-	"This control plane is not running! (state={{.state}})": "",
-	"This driver does not yet work on your architecture. Maybe try --driver=none": "",
-	"This is a known issue with BTRFS storage driver, there is a workaround, please checkout the issue on GitHub": "",
+	"This control plane is not running! (state={{.state}})": "このコントロールプレーンは動作していません! (state={{.state}})",
+	"This driver does not yet work on your architecture. Maybe try --driver=none": "このドライバーはあなたのアーキテクチャではまだ機能しません。もしかしたら、--driver=none を試してみてください",
+	"This is a known issue with BTRFS storage driver, there is a workaround, please checkout the issue on GitHub": "これは BTRFS ストレージドライバーの既知のバグです (回避策があります)。GitHub の issue を確認してください",
```
Comment (atoato88) [inline translations/ja.json:710]:
```suggestion
	"This is a known issue with BTRFS storage driver, there is a workaround, please checkout the issue on GitHub": "これは BTRFS ストレージドライバーの既知の問題です (回避策があります)。GitHub の issue を確認してください",
```
In translations file, the word "bug" is used to represent just bug, and here "issue" is used, not "bug".
I think `問題` is better translation.

**Comment by atoato88** [comment]
@yosshy 
I left just one comment. Could you check it?

**Comment by yosshy** [comment]
@atoato88 thank you for your review. I accepted your fix.

**Comment by atoato88** [comment]
/lgtm
Thank you to update!
It's OK as Japanese review 😀

**Comment by yosshy** [comment]
/assign @prezha
Would you merge it?

**Review by spowelljr — APPROVED** [review]

**Pull request merged**